### DISCUSSION TITLE
Angular V12 Support

### DISCUSSION THREAD

**Discussion by rhllor** [ocombe/ng2-translate#1310] [Q&A] [answered]
Hi, 
In documentation I can't find any information. 
The new version of Angular is supported from library?

The peer dependency want @angular/compiler@>=2.3.1 <12.0.0, but i need to work with the last version of Angular and i18n standard is not good for me. 

Thanks

**Comment by ocombe** [comment, marked as answer]
Hello,

it works with angular 12, the peer dependency is >=10: https://github.com/ngx-translate/core/blob/master/projects/ngx-translate/core/package.json#L22

**Reply by rhllor** [reply]
Thanks, 
I resolve my problems with this library. I Love it.

**Reply by j3gb3rt** [reply]
Can you update the compatibility chart in the readme to reflect this?

**Reply by ocombe** [reply]
Done, thanks for the suggestion.
